Makefile.toml

```diff
@@ -23,7 +16,7 @@ dependencies = ["lint", "clippy", "check", "test"]
 
 [tasks.lint]
 description = "Lint code style (formatting, typos, docs, markdown)"
-dependencies = ["lint-format", "lint-typos", "lint-docs", "lint-markdown"]
+dependencies = ["lint-format", "lint-typos", "lint-docs"]
```

Comment:
Removed for now. This is unrelated, but will re-enable when this is reconfigured to correctly ignore the generated parts.